packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1553,4 +1553,20 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
     }
     return descendantText;
   }
+
+  /**
+  * Get the aria autocomplete value for the Combobox
+  * @returns 'inline' if auto-complete automatically dynamic, 'both' if we have a list of possible values to pick from and can
+  * dynamically populate input, and 'none' if auto-complete is not enabled as we can't give user inputs.
+  */
+  private _getAriaAutoCompleteValue(): string {
+    let autoComplete = !this.props.disabled && this.props.autoComplete;
```

Comment:
doesn't the autoComplete code need to compare against 'on' since it's not a boolean? The current code will be true if this.props.autoComplete ===  'off'